Libplanet/Action/IAction.cs

```diff
@@ -313,6 +314,20 @@ public interface IAction
         /// </param>
         void Render(IActionContext context, IAccountStateDelta nextStates);
 
+        /// <summary>
+        /// Does things that should be doen right after this action is
+        /// spread to the network or is &#x201c;confirmed&#x201d; (kind of)
+        /// by each peer node.
```

Comment:
How about explaining this method like the following way:

```suggestion
        /// Does the similar things to <see cref="Render(IActionContext, IAccountStateDelta)"/>,
        /// except that this method is invoked when see cref="Execute(IActionContext)"/> method
        /// has terminated with an exception.
```